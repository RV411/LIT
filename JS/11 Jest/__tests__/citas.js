import Citas  from "../js/classes/Citas.js";

describe('Probar la clase de Citas',()=>{

    const citas=new Citas();

    test('Agregar una nueva cita',()=>{
        const citaObj = {
            mascota: 'Hook',
            propietario: 'Juan',
            telefono: '19030913',
            fecha: '10-12-2020',
            hora:'10:30',
            sintomas: 'Solo duerme'
        };

        citaObj.id=Date.now();
        citas.agregarCita(citaObj);

        //Prueba
        expect(citas).toMatchSnapshot();
    });
});