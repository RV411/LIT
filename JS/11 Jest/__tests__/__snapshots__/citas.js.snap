// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probar la clase de Citas Agregar una nueva cita 1`] = `
Citas {
  "citas": [
    {
      "fecha": "10-12-2020",
      "hora": "10:30",
      "id": 1671048572784,
      "mascota": "Hook",
      "propietario": "Juan",
      "sintomas": "Solo duerme",
      "telefono": "19030913",
    },
  ],
}
`;
